modules/standard/Map.chpl

```diff
@@ -759,108 +791,108 @@ module Map {
   }
 
   /* Returns a new map containing the keys and values in either a or b. */
-  operator map.+(a: map(?keyType, ?valueType, ?parSafe),
-                 b: map(keyType, valueType, parSafe)) {
+  operator map.+(a: map(?keyType, ?valueType, ?),
+                 b: map(keyType, valueType, ?)) {
     return a | b;
   }
 
   /*
     Sets the left-hand side map to contain the keys and values in either
     a or b.
    */
-  operator map.+=(ref a: map(?keyType, ?valueType, ?parSafe),
-                  b: map(keyType, valueType, parSafe)) {
+  operator map.+=(ref a: map(?keyType, ?valueType, ?),
+                      b: map(keyType, valueType, ?)) {
```

Comment:
I'd prefer having a single indentation strategy when splitting lines. Other places line the second line up right after the opening parenthesis.